pkgs/by-name/pa/papers/package.nix

```diff
@@ -0,0 +1,161 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+, meson
+, ninja
+, pkg-config
+, appstream
+, desktop-file-utils
+, gtk4
+, glib
+, pango
+, atk
+, gdk-pixbuf
+, shared-mime-info
+, itstool
+, gnome
+, poppler
+, ghostscriptX
+, djvulibre
+, libspectre
+, libarchive
+, libsecret
+, wrapGAppsHook
+, librsvg
+, gobject-introspection
+, yelp-tools
+, gspell
+, gsettings-desktop-schemas
+, dbus
+, gst_all_1
+, gi-docgen
+, supportMultimedia ? true # PDF multimedia
+, libgxps
+, supportXPS ? true # Open XML Paper Specification via libgxps
+, withLibsecret ? true
+, libadwaita
+, exempi
+, cargo
+, rustPlatform
+}:
+
+stdenv.mkDerivation rec {
+  pname = "papers";
+  version = "unstable-2024-3-27";
```

Comment:
It should be `45.0-unstable-2024-03-27`.

Reply:
Why 45 rather than 46 ?